app/services/transaction_search/date_range_searcher.rb

```diff
@@ -8,7 +8,11 @@ class DateRangeSearcher < BaseSearcher
     attr_reader :date_range_field
 
     def self.options
-      FIELDS.map { |field| [I18n.t(field, scope: "admin.transaction_search.date_range_fields"), field] }
+      if @billing_context
```

Comment:
I'm guessing there's a better way to do this than passing the `billing_context` flag around, but I'm not sure what it would be. Also not crazy about the name of that flag.

Reply:
Looks like we had some of the same thoughts. If we just don't have this option, what's the worst case? That the new/inprocess has a couple options that will result in 0 results. I'm kind of ok due to what the effort would likely be to get rid of them.